src/main/resources/assets/gcys/lang/en_us.lang

```diff
@@ -484,6 +484,10 @@ item.material.oreprefix.boule=%s Boule
 
 # Metaitems
 
+# mold
+
+metaitem.shape.mold.chunk.name=Mold (Chunks)
+
```

Comment:
This change actually needs to be made in upstream CEu, in the ore work branch. Do you mind removing it for now?